src/libraries/Microsoft.Extensions.Logging.Console/src/Microsoft.Extensions.Logging.Console.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>$(NetcoreAppCurrent)-Windows_NT;$(NetcoreAppCurrent);netstandard2.0-Windows_NT;netstandard2.0</TargetFrameworks>
```

Comment:
Are you doing this just so you can call the Console Mode on windows?

If so, you should just produce a single`$(NetCoreAppCurrent);` binary, and check `if (RuntimeInformation.IsOSPlatform(Windows))` instead. I'm not entirely sure we can ship OOB packages like this, with different assemblies for windows vs. non.

Reply:
I agree we should avoid cross compiling at least for the sake of the package size. runtime checks looks reasonable to me in this case.

Reply:
>  I'm not entirely sure we can ship OOB packages like this, with different assemblies for windows vs. non.

If there is not a runtime assembly it will use `lib/net5.0` for that runtime right?

Yeah, she needs `Interop.GetConsoleMode.cs` but I'm not sure how many file we would need to pull in to make it compile in `Unix`.

Reply:
looks she doesn't need anything else more than defining `Libraries.Kernel32`

Reply:
If that’s it then yeah I would also prefer runtime checks.

Reply:
💯 we should try to keep Microsoft.Extensions.* RID-agnostic